man/subscription-manager.8

```diff
@@ -250,9 +250,15 @@ Sets the preferred service level to use with subscriptions added to the system.
 
 .TP
 .B --force
-Registers the system even if it is already registered. Normally, any register operations will fail if the system is already registered. With
+Normally, register operations will fail if the system is already registered.  With
```

Comment:
since this is describing something a fact, something that happens every time, shouldn't "will fail" better be stated as "fails"?

Reply:
I don't really like the phrase "Normally, register operations". What do you think of replacing the first two sentences with this...  "When the system is already registered, a new attempt to register will fail with a message reminding the user that the system is already registered. However, passing the --force option will implicitly attempt to unregister the system first."   Then continue with the "Beware that..."

Reply:
sounds good to me, will make the change